dumb-init.c

```diff
@@ -128,6 +128,16 @@ int main(int argc, char *argv[]) {
         }
 
         execvp(argv[1], &argv[1]);
+
+        // if this point is reached, exec failed, so we should exit nonzero
+        fprintf(
+            stderr,
+            "Failed to launch process '%s' (errno %d: %s)\n",
```

Comment:
keeping closer to the other unix commands:

'dumb-init: %s: %s', argv[1], strerror(errno)